charts/templates/FlinkCluster.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if not .Values.onlyApplication }}
```

Comment:
To make troubleshooting much easier, I recommend using one Flink cluster per Flink app.
See example https://github.com/pravega/video-samples/blob/master/charts/videoprocessor/templates/MultiVideoGridJob.yaml.